tests/iio_stresstest.c

```diff
@@ -385,15 +385,18 @@ int main(int argc, char **argv)
 			break;
 		case 'b':
 			info.arg_index += 2;
-			info.buffer_size = atoi(info.argv[info.arg_index]);
+			info.buffer_size = strtol(info.argv[info.arg_index], NULL, 10);
+			/* Max 4M */
+			if (info.buffer_size > (1024 * 1024 * 4))
+				info.buffer_size = 1024 * 1024 * 4;
```

Comment:
this max looks like slightly new behavior;
but it's fine to leave it here

Reply:
yes, it was complaining that the value was tainted (not checked), so check it....